src/utils/hints/contracts.js

```diff
@@ -0,0 +1,6 @@
+export const contractsHints = {
+  contractId: 'A Smart Contract instance is an entity living on the block chain or in a state channel. Each instance has an address that can be used to call its entrypoints, either from another Smart Contract or in a ContractCallTx.',
+  contractCreated: 'Date, time and block height when the Smart Contract was created.',
+  contractTxHash: 'The transaction of type ContractCreateTx that created the Smart Contract on the blockchain.',
```

Comment:
proposal:
- `The transaction hash of the ContractCreateTx that was executed to create the Smart Contract on the blockchain.`

Reply:
Proposal accepted.